validation/src/test/scala/hmda/validation/rules/lar/quality/Q046Spec.scala

```diff
@@ -0,0 +1,33 @@
+package hmda.validation.rules.lar.quality
+
+import hmda.model.fi.lar.LoanApplicationRegister
+import hmda.validation.rules.EditCheck
+import hmda.validation.rules.lar.{ BadValueUtils, LarEditCheckSpec }
+
+class Q046Spec extends LarEditCheckSpec with BadValueUtils {
+
+  property("Valid if loan purpose not 1") {
+    forAll(larGen, intOtherThan(1)) { (lar, x) =>
+      val newLoan = lar.loan.copy(purpose = x)
+      val newLar = lar.copy(loan = newLoan)
+      newLar.mustPass
+    }
+  }
+
+  property("Valid if hoepa not 1") {
+    forAll(larGen, intOtherThan(1)) { (lar, x) =>
+      val newLar = lar.copy(hoepaStatus = x)
+      newLar.mustPass
+    }
+  }
+
+  property("Invalid if hopea 1 and loan purpose 1") {
```

Comment:
Tiny correction: In this spec, as well as Q052Spec and Q064Spec, there's a spelling error in HOEPA (it's spelled "hopea" in a few property descriptions)